ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -245,7 +250,11 @@ export class EvalCommand extends Command<EvalCommandReturn> {
             ConfigSchema.CFGU.VALIDATORS.valueOptions(cfgu, evaluatedValue);
             ConfigSchema.CFGU.VALIDATORS.valueType(cfgu, evaluatedValue);
           } catch (error) {
-            throw error?.appendScope?.(errorScope) ?? error;
+            let err = error?.appendScope?.(errorScope) ?? error;
+            if (current.result.origin === EvaluatedConfigOrigin.StoreSet) {
+              err = new ConfigStoreError('invalid config value', err.message, err?.scope ?? errorScope);
+            }
```

Comment:
In my opinion, this shouldn't result in a store error. Let's get product opinion on this.

Reply:
Ran agrees

Reply:
Revert